wondrous-app/components/Common/CornerWidget/Component.tsx

```diff
@@ -0,0 +1,122 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import CloseModalIcon from 'components/Icons/closeModal';
+import get from 'lodash/get';
+import { useRouter } from 'next/router';
+
+import palette from 'theme/palette';
+import typography from 'theme/typography';
+import { ENTITIES_TYPES } from 'utils/constants';
+
+import { CornerWidgetComponentType } from './types';
```

Comment:
1. you don't need to have the types in a separate file, as you don't reuse them. 
2. just name it `Props`